test/integration/cloud/serverspec/pgd_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'serverspec'
```

Comment:
I noticed that this file is functionally equivalent to the others named `pgd_spec.rb` in other suites. It'd be nice if we could refactor so that only one copy of this file exists in any one place.

Could you try moving this file outside of `test/integration/cloud`, and then symlinking `test/integration/cloud/serverspec/pgd_spec.rb` to it? My hope is that busser will still copy it over and run the tests correctly, but I worry that it will not do so as I hope. If it did, we could easily refactor these down to one file.

If it doesn't work that way then I'm not sure how to easily refactor these three files. @armiller might have an idea.